versioning/revisions.rst

```diff
@@ -76,6 +76,58 @@ Take into account that it changes the default Conan behavior. e.g:
     - If you generate and upload N binary packages for a recipe with a given revision, then if you modify the recipe, and thus the recipe
       revision, you need to build and upload N new binaries matching that new recipe revision.
 
+
+GIT and Line Endings on Windows
+-------------------------------
+
+There is one very common problem that users encounter after enabling revisions which all users should be aware of. 
+This issue occurs when all of the following conditions are true:
+
+ - Using GIT to retrieve sources with default settings
+ - Using Continuous Integration services
+ - Building on multiple platforms
+ - One of the target platforms is Windows
+
+When cloning a repository on a Windows machine, the GIT client will replace all of the line endings on all of the 
+files with @CRLF character. The default line ending outside of Windows is @LF.  
+
+**Problem**
+
+As a result, when Conan does export of an otherwise identical GIT repository/commit on a Windows and a Non-Windows build machine,
+the Conan revisions will be different between the two. This has a very unfortunate consequence when two such packages are then 
+uploaded to a Conan repository (as is very common in continuous integration workflows). The net consequence of two such uploads, 
+is that the next time `conan install` is run for that package, the "latest" revision will only have the binaries for one platform
+or the other (whichever CI job finished and uploaded last). If `conan install` is run for the other platform, it will receive
+the following error:
+    
+    ERROR: Missing prebuilt package for <package_ref>
+
+**Workaround**
+
+The problem is unfortunately something that is external to Conan, so we cannot provide a general-purpose fix. We
+can only explain the situation as we have done here and suggest a few ways to address the problem.  The most 
+straightforward and reasonable solution is to prevent GIT for windows from replacing the line-endings on the Windows 
+build machines. This avoids the problem completely by ensuring that the Conan Revisions are the same on Windows and 
+Non-Windows machines. There are several ways to achieve the goal. 
+
+For users who can add files at the repository level, a file named `.gitconfig` can be added which contains the following:
```

Comment:
I am not sure this works: https://git-scm.com/docs/git-config => $GIT_DIR/config

The usual workaround is ``.gitattributes`` file.